config/initializers/committee.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'committee'
+
+Rails.application.config.middleware.use(
+  Committee::Middleware::RequestValidation,
+  schema_path: Rails.public_path.join('openapi.json').to_s,
+  strict_reference_validation: true
```

Comment:
I'd expect a 422 rather than a 400 if the data doesn't conform to the schema. I think we can do that by adding `error_class: Common::Exceptions::ValidationErrors` here.